apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/StandardCacheHeaders.java

```diff
@@ -0,0 +1,14 @@
+package com.apollographql.apollo.cache.normalized;
+
+/**
+ * A collection of "standardized" cache headers that an implementation {@link NormalizedCache}
+ * is recommended to respect.
+ */
+public class StandardCacheHeaders {
+
+  /**
+   * Records from this request should not be stored in the {@link NormalizedCache}.
+   * This does **not** specify that a request should not be read from a cache.
+   */
+  public static final String DO_NOT_CACHE = "do-not-cache";
```

Comment:
Shouldn't it be part of CacheControl? It becomes confusing that we have 2 ways how to control cache. I thought the CacheHeaders should act as a helper to CacheControl. (like TTL) but not control the cache itself.